pkg/api/vlabs/validate.go

```diff
@@ -1828,6 +1826,10 @@ func (a *Properties) validateContainerRuntime(isUpdate bool) error {
 				return errors.Errorf("WindowsSdnPluginURL must be provided when using Windows with ContainerRuntime=containerd and networkPlugin=kubenet")
 			}
 		}
+
+		if a.OrchestratorProfile.KubernetesConfig.NetworkPlugin == NetworkPluginAntrea {
+			return errors.Errorf("networkPlugin antrea for windows is not supported with ContainerRuntime=containerd")
```

Comment:
Is support for containerd in the roadmap?

Reply:
We havent planned for containerd support yet.